vector/src/main/java/im/vector/app/features/roommemberprofile/RoomMemberProfileViewModel.kt

```diff
@@ -121,6 +128,24 @@ class RoomMemberProfileViewModel @AssistedInject constructor(@Assisted private v
                 }
     }
 
+    private fun observeAccountData() {
+        session.flow()
+                .liveUserAccountData(UserAccountDataTypes.TYPE_OVERRIDE_COLORS)
+                .unwrap()
+                .map { it.content.toModel<Map<String, String>>() }
+                .map { userColorAccountDataContent ->
```

Comment:
nit: could merge these maps 

```kotlin
.map { 
  val userColorAccountDataContent = it.content.toModel<Map<String, String>>()
  userColorAccountDataContent?.get(initialState.userId)
}
```

Reply:
Due to rework, thanks, will update

Reply:
51c9c2f032a7709d18b56bbfb35def25e3d08a24